test/Api.IntegrationTest/SecretsManager/Controllers/SecretsControllerTests.cs

```diff
@@ -1003,20 +1016,45 @@ private async Task UpdateServiceAccountRevisionAsync(Guid serviceAccountId, Date
 
         var currentOrganizationUser = orgAdminUser;
 
-        if (permissionType == PermissionType.RunAsUserWithPermission)
+        switch (permissionType)
         {
-            var (email, orgUser) = await _organizationHelper.CreateNewUser(OrganizationUserType.User, true);
-            await _loginHelper.LoginAsync(email);
+            case PermissionType.RunAsUserWithPermission:
+                {
+                    var (email, orgUser) = await _organizationHelper.CreateNewUser(OrganizationUserType.User, true);
+                    await _loginHelper.LoginAsync(email);
 
-            var accessPolicies = new List<BaseAccessPolicy>
-            {
-                new UserProjectAccessPolicy
+                    var accessPolicies = new List<BaseAccessPolicy>
                 {
-                    GrantedProjectId = project.Id, OrganizationUserId = orgUser.Id, Read = true, Write = true
+                    new UserProjectAccessPolicy
+                    {
+                        GrantedProjectId = project.Id,
+                        OrganizationUserId = orgUser.Id,
+                        Read = true,
+                        Write = true
+                    }
+                };
```

Comment:
🤔 this does look like a formatting issue. Can you check ?

Reply:
This is `dotnet format` making any manual changes will fail the CI formatting check. 

The only other option is to swap to if statements. 

I went ahead and did that for readability https://github.com/bitwarden/server/pull/4160/commits/f5e160956ad7ec75c680d67d0bd3784617cc9d5b